scipy/sparse/linalg/_interface.py

```diff
@@ -703,6 +704,12 @@ def __init__(self, A, alpha):
             raise ValueError('LinearOperator expected as A')
         if not np.isscalar(alpha):
             raise ValueError('scalar expected as alpha')
+        if isinstance(A, type(self)):
+            A, alpha_original = A.args
```

Comment:
This should IMO be

```suggestion
        if isinstance(A, _ScaledLinearOperator):
            A, alpha_original = A.args
```
for clarity. LinearOperator does not have an `args` member.